recipe/meta.yaml

```
@@ -16,7 +16,7 @@ source:
     sha256: aedbbc280610875b83e246f05957369f84acf9abdb86b884a9ff2f5dd65e8ffb
     folder: xz-LICENSE-2
   - url: https://gitlab.com/cznic/mathutil/-/raw/master/LICENSE
-    sha256: 5c640056d513f276d56635af7a0bed81cb7eac02a31b6e56d40c416557a8d8ce
+    sha256: bfa9bf72a72ca009fd62a8f84fca3dca67e51d93af96352723646599898b6cf5
```

@ytausch any idea why this always changes the license sha? this was already the case on the last pr

```suggestion
    sha256: 5c640056d513f276d56635af7a0bed81cb7eac02a31b6e56d40c416557a8d8ce
```

---

After looking into this, I found that the bot actually does the right thing and the hash mismatch is actually a bug in `conda-build` (!)

`bfa9bf72a72ca009fd62a8f84fca3dca67e51d93af96352723646599898b6cf5 ` is the correct hash of the mathutil license, and `5c640056d513f276d56635af7a0bed81cb7eac02a31b6e56d40c416557a8d8ce` (what conda-build wants here) is the hash of xz from a few lines above.

I'll open a PR to migrate this to rattler build.